src/box/lua/schema.lua

```diff
@@ -2365,6 +2378,7 @@ end
 
 base_index_mt.select_luac = function(index, key, opts)
     check_index_arg(index, 'select')
+    check_select_args(index, key, opts)
```

Comment:
We don't care about unlimited select calls over iproto in this implementation, correct? I don't see how it differs from the call from Lua. I asked the product team about this and will return back here tomorrow.

Reply:
@NeraverinTarantool said that the product team considered it as not so important problem.